builtin/builtin.go

```diff
@@ -1,13 +0,0 @@
-// Package builtin contains all packages related to Flux built-ins are imported and initialized.
-// This should only be imported from main or test packages.
-// It is a mistake to import it from any other package.
```

Comment:
Could you add a similar comment to the new `static` package?

Reply:
Oh yes thanks for catching that.